examples/Cargo.toml

```diff
@@ -1,17 +0,0 @@
-[package]
```

Comment:
You can still do that... use `cargo run --example simple` and it should just work. Or, if you want to keep it as a separate crate, it should be a directory nested inside `examples` that contains a `main.rs` and the `Cargo.toml`. The way it was set up resulted in `cargo check --all-targets` failure, because `examples/simple.rs` is actually part of the top-level crate, but the top-level crate did not have the necessary dependencies.

Reply:
Thanks for explanation, looks good I'll merge it now